src/lib/comms/sol-oic-client.c

```diff
@@ -477,6 +491,8 @@ client_get_info(struct sol_oic_client *client,
     int r = 0;
 
     ctx = sol_util_memdup(&(struct server_info_ctx) {
+            .base.reentrant.in_use = false,
+            .base.reentrant.delete_me = false,
```

Comment:
hum... my proposal to `sol_reentrant_init()` couldn't be used here. However, by default non-specified members default to zero, so false.